demo/src/App.js

```diff
@@ -100,6 +103,13 @@ export default class App extends PureComponent {
         </div>
       );
     });
+    const demos = Object.keys(sources)
+      .map(source => (
+        <a href={`?demo=${source}`} key={source}>
+          {source}
+        </a>
+      ))
+      .reduce((prev, curr) => [prev, ' / ', curr]);
```

Comment:
Can we only import `sources` once?

Reply:
Confused: You don't mean `import` like the keyword?

Reply:
This was more in reference to my overall comment.  Importing in both `App.js` and `index.js` feels a bit odd.